src/sr_od/operations/enter_game/open_game.py

```
@@ -16,6 +38,36 @@ def __init__(self, ctx: SrContext):
+        json_str = f'{{"width":{width},"height":{height},"isFullScreen":{str(is_full_screen).lower()}}}\0'
+        values: dict[str, tuple[bytes | int, int]] = {
+            _REG_GRAPHICS_SETTINGS: (json_str.encode('ascii'), winreg.REG_BINARY),
+            _REG_SCREEN_WIDTH: (width, winreg.REG_DWORD),
+            _REG_SCREEN_HEIGHT: (height, winreg.REG_DWORD),
+            _REG_FULLSCREEN_MODE: (fullscreen_mode, winreg.REG_DWORD),
+        }
+
+        global _resolution_patch
+        patch = RegistryPatch(subkey)
+        if not patch.backup_and_set(values):
+            log.warning('注册表分辨率设置失败，将继续使用当前设置')
```

Ruff (0.15.2)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)